notebooks/RKI_GER_COVID19_DASHBOARD.ipynb

```diff
@@ -139,15 +140,30 @@
     "    \"last_update\": \"last_update\",   \n",
     "}\n",
     "df.rename(columns=column_name_map, inplace=True)\n",
+    "\n",
+    "# map district types\n",
+    "district_type_map = {\n",
+    "    \"Landkreis\" :\"Rural District\",\n",
+    "    \"Kreis\": \"Rural District\",\n",
+    "    \"Kreisfreie Stadt\": \"District-free City\",\n",
+    "    \"Stadtkreis\": \"District-free City\"\n",
+    "}\n",
+    "\n",
+    "df['district_type'] = df['district_type'].apply(lambda d: district_type_map.get(d))\n",
+    "\n",
+    "# set district id as index\n",
     "df.set_index(\"district_id\")\n",
+    "\n",
+    "# select columns by column_name_map\n",
     "df = df.loc[:, tuple(column_name_map.values())]\n",
     "\n",
-    "# last_update -> epoch\n",
-    "def resolve_last_update(row):\n",
-    "    date_regex = re.search(r'(\\d{2})\\.(\\d{2})\\.(\\d{4}), (\\d{2}):(\\d{2})', row)\n",
-    "    return int(datetime.fromisoformat(f'{date_regex.group(3)}-{date_regex.group(2)}-{date_regex.group(1)}T{date_regex.group(4)}:{date_regex.group(5)}+02:00').timestamp()) * 1000\n",
+    "# parse last_update\n",
+    "df['last_update'] = df['last_update'].apply(lambda row: pd.to_datetime(row, format=\"%d.%m.%Y, %H:%M Uhr\").tz_localize(\"Europe/Berlin\"))\n",
     "\n",
-    "df['last_update'] = list(map(lambda row: resolve_last_update(row), df['last_update']))"
+    "# set ISO3166_1, ISO3166_2 fields\n",
+    "df['ISO3166_1'] = 'DE'\n",
+    "subdivisions = {k.name: k.code.replace(\"DE-\", \"\") for k in pycountry.subdivisions.get(country_code=\"DE\")}\n",
+    "df['ISO3166_2'] = df['state'].apply(lambda c: subdivisions.get(c))"
    ]
```

Comment:
please add a standard `LAST_UPDATED_DATE` field with local UTC time (same as in other notebooks)